pkg/mod/executor_test.go

```diff
@@ -260,8 +262,7 @@ func TestDefExecutor_WorkerDo(t *testing.T) {
 				Params: map[string]interface{}{
 					"field1": 1,
 				},
-				Status: entity.TaskInstanceStatusFailed,
-				Reason: "get task params from task instance failed: json unmarshal: json: cannot unmarshal number into Go struct field TestParam.field1 of type string",
+				Status: entity.TaskInstanceStatusSuccess,
```

Comment:
I think we can modified the test case to return a error , i.g.  give a text to a int filed and rename this case